DOC-3243: Dialogs did not retain focus when using the block/unblock API. (#4018)
tinymce/tinymce-docs: modules/ROOT/pages/8.4.0-release-notes.adoc

// CCFR here.

=== Dialogs did not retain focus when using the block/unblock API.
// #TINY-12918

> Previously, when a dialog was blocked (for example, while loading) and then unblocked, the loading spinner disappeared without automatically focusing the first focusable element in the dialog. Focus was lost from the editor entirely. When using an inline editor, this caused the dialog and the rest of the editor UI to disappear when the dialog lost focus.

Reviewer: ```suggestion
Previously, when a dialog was blocked (for example, while loading) and then unblocked, the loading spinner disappeared without automatically focusing any other element. As a result, focus escaped the editor entirely. When using an inline editor, this caused the dialog and the rest of the editor UI to disappear.
```
